python_modules/dagster/dagster/_core/definitions/metadata/table.py

```diff
@@ -259,3 +260,24 @@ def __new__(
 
 
 _DEFAULT_TABLE_COLUMN_CONSTRAINTS = TableColumnConstraints()
+
+
+# ###########################
+# ##### TABLE COLUMN LINEAGE
+# ###########################
+
+
+@whitelist_for_serdes
+class AssetColumnDep(NamedTuple):
```

Comment:
This should probably be named `TableColumnDep`. But calling it `TableColumnDep` felt weird since `TableColumn` is already taken and the existing `TableColumn` doesn't have an existing reference to an asset.

Open to other suggestions.